### ISSUE TITLE
Document `type`

### ISSUE THREAD

**Issue by strugee** [dokku/docker-container-healthchecker#168]
### Description of feature

Both the README and

**Comment by josegonzalez** [comment]
Yeah it refers to that, but the docker-local scheduler only supports the `startup` type. All are supported for the `k3s` scheduler.

In the future, the custom proxy plugins might enable some sort of support for the `readiness` type so that they don't proxy to things that can't be proxied to yet. We might also finally have a proper daemon that does healthchecks for the docker-local scheduler and handles restarts appropriately.

**Comment by drbraden** [comment]
Is there a way to set the Docker "Test" command for the container when using a non-web process?  For example, I have a queue worker that doesn't serve web requests.  If I do a `docker inspect` the "HealthCheck" looks like this:
```
"Healthcheck": {
     "Test": [
         "CMD",
         "sh",
         "-c",
         "curl --insecure --silent --location --show-error --fail http://localhost:8080$HEALTHCHECK_PATH || exit 1"
     ],
     "Interval": 5000000000,
     "Timeout": 3000000000,
     "Retries": 3
},
```

So a `docker ps` will always list the STATUS as something like "Up 8 minutes (unhealthy)".

I just tried adding a "liveness" type to my `app.json` in the hope that it would affect the container's healthcheck command but it's still set to the curl command test.

**Comment by josegonzalez** [comment]
This plugin doesn't change any `HEALTHCHECK` directives in a dockerfile. Also, for docker-local, only the `startup` type, so adding a `liveness` or `readiness` check there wouldn't do anything unless the scheduler allows for it.

**Comment by drbraden** [comment]
Ah, that was the clue I needed!  It's being set in a base image I'm pulling.  That explains why none of my other deploys were showing "unhealthy" for non-web processes, as this is my first deploy with this particular base image (the others just rely on the standard buildpacks which don't set healthcheck).

Got it, thank you!
